pages/people.md

```diff
@@ -47,6 +47,7 @@ based on commits.
 | [Shan Nawaz](https://github.com/shannawaz)                  | [Microsoft][msft]                   | [ET][et]   |
 | [Sri Kailash](https://github.com/srikailash)                | [Booking.com](https://booking.com)  | [CEST][cst]|
 | [Tanisha Faulkner](https://github.com/plainolneesh)         | [ASRC Federal](https://www.asrc.com)| [ET][et]   |
+| [Bhuvan Mysore Sreenivas](https://github.com/flyingcanopy)  |                                     | [IST][ist] |
```

Comment:
@flyingcanopy we try to keep this list in alphabetical order.  Would you mind moving your entry between Amal and Eric?

Reply:
@keith-turner done.